apps/ledger-live-mobile/src/components/BleDevicePairingFlow/index.tsx

```diff
@@ -11,6 +13,7 @@ import type { BleDevicePairingProps } from "./BleDevicePairing";
 import { track } from "../../analytics";
 import { NavigationHeaderBackButton } from "../NavigationHeaderBackButton";
 import { NavigationHeaderCloseButton } from "../NavigationHeaderCloseButton";
+import UnlockDeviceDrawer from "../../screens/SyncOnboarding/UnlockDeviceDrawer";
```

Comment:
If this `UnlockDeviceDrawer` is re-used we should either copy it here in this flow (in the `components/BleDevicePairingFlow`) or move it at the root of our `components` folder

I would go with the moving it to `components/` so it can be re-used